admin-ui/app/utils/formUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { isEqual } from 'lodash'
+
+type Primitive = string | number | boolean | null | undefined
+
```

Comment:
The name of the file should follow the convention and start with upper case. e.g. FormUtils.ts